app/src/check.js

```diff
@@ -46,7 +46,7 @@ class Check extends React.Component {
   nestItems (items) {
     let prevIndex = null
     return items.reduce((matchItems, item) => {
-      if (item.matched === 'True') {
+      if (item.matched === 'True' || item.matched === null) {
```

Comment:
Maybe we should eventually change the name of this from "matched" to "exact_match"